opencl-aot/source/main.cpp

```diff
@@ -484,7 +572,7 @@ int main(int Argc, char *Argv[]) {
     return OPENCL_AOT_FAILED_TO_CREATE_ELF;
   }
   std::cout << "OpenCL program binary file was successfully created: "
```

Comment:
If during --cmd=compile, we will get OpenCL program object, this message should be changed